src/__stories__/callout-story.tsx

```diff
@@ -16,19 +16,12 @@ type Args = {
     title: string;
     description: string;
     actions: string;
-    withIcon: boolean;
-    isClosable: boolean;
+    icon: boolean;
```

Comment:
should we use "Asset" in this case? We have pending to change "icon" prop to "asset". Maybe we can go ahead with asset in stories? 

https://github.com/Telefonica/mistica-web/issues/626
![image](https://github.com/user-attachments/assets/97f842c3-89e4-422d-a05e-f5e8f6ce47f9)

Reply:
Fixed in Callout, Dialog and Feedback